install.py

```diff
@@ -747,14 +743,18 @@ def driver():
         dest="legion_dir",
         required=False,
         default=None,
-        help="Path to an existing Legion build directory.",
+        help="Path to an existing Legion build directory. A recent checkout "
+        "of the control_replication branch is required, configured with "
```

Comment:
I don't think the `control_replication` branch would be an actual requirement?

See above. We should enforce these requirements in the cmake rather than just documenting them here.

Reply:
Unfortunately the Legion API has significant changes between the `master` branch (which is what actually gets released) and the `control_replication` branch, which we depend on. AFAIK there is no exported configuration option that we can use to check whether this is a `master` or `control_replication` Legion build. This is orthogonal to the fact that we require `Legion_REDOP_COMPLEX`.

Perhaps we could use some other trick to confirm that we're on a `control_replication` build, but this still doesn't solve the bigger problem of version compatibility between Legion and Legate (Legion adds an API extension, we start depending on that, but there is no version tagging on the `control_replication` tree that we can use to confirm that the linked Legion is new enough).

I think the best long-term solution here is to wait for the merge of `control_replication` into `master`, and then we can start depending on specific released versions of Legion. But maybe @lightsighter has another suggestion.

Reply:
sounds good